### PR TITLE
feat(Spanner): Add bit reversed sequence samples

### PR DESCRIPTION
**Pull request by ajupazhamayil**

### CODE REVIEW

**Review by bshaffer — COMMENTED** [review]
The error here is from the Spanner API:

```
Google\Cloud\Core\Exception\NotFoundException: {
    "message": "Sequence is not supported.",
    "code": 12,
    "status": "UNIMPLEMENTED",
    "details": []
}
```

This would indicate that the feature being requested is not enabled on the projects we are using.

**Review by rajatbhatta — APPROVED** [review]
LGTM, apart from a couple of nits.

File: spanner/src/create_sequence.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        "('Alice'), ('David'), ('Marc') THEN RETURN CustomerId"
+    )->rows(Result::RETURN_ASSOCIATIVE);
+
+    $insertCount = 0;
+    foreach ($rows as $row) {
+        printf('Inserted customer record with CustomerId: %d %s',
+            $row['CustomerId'],
+            PHP_EOL
+        );
+        $insertCount++;
```
[the remark this answers is not in the document]
Comment (ajupazhamayil) [reply spanner/src/create_sequence.php:75]: We have `stats` function in the `Result`. We will make use of that. Thank you for the suggestion.

[the remark this answers is not in the document]
Comment (ajupazhamayil) [reply spanner/src/create_sequence.php]: Done

File: spanner/src/create_sequence.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    ): void {
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+    $transaction = $database->transaction();
+
+    $operation = $database->updateDdlBatch([
+        "CREATE SEQUENCE Seq OPTIONS (sequence_kind = 'bit_reversed_positive')",
+        'CREATE TABLE Customers (CustomerId INT64 DEFAULT (GET_NEXT_SEQUENCE_VALUE(' .
+        "'Seq')), CustomerName STRING(1024)) PRIMARY KEY (CustomerId)"
```
[the remark this answers is not in the document]
Comment (ajupazhamayil) [reply spanner/src/create_sequence.php]: I see this is the change when it came to production. Done, thank you!

**Review by yash30201 — APPROVED** [review]

File: spanner/src/create_sequence.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (rajatbhatta) [reply spanner/src/create_sequence.php]: Yes

**Pull request merged**